submissions/Oleksii_Drohachov/Tiny_JS_world/index.js

```diff
@@ -0,0 +1,72 @@
+
+catsPhrase = 'meow-meow skinbag';
+
+const tinyWorldInhabitants = [
+   {
+      species: 'dog',
+      name: 'Barbos',
+      gender: 'male',
+      legs: 4,
+      hands: 0,
+      saying: 'gav-gav kaje pes',
+      friends: ['definitely everybody in the world !']
+   },
+   {
+      species: 'cat',
+      name: 'Sonya',
+      gender: 'female',
+      legs: 4,
+      hands: 0,
+      saying : catsPhrase,
+      friends: ['definitely nobody except her own slaves']
+   },
+   {
+      species: 'human',
+      name: 'Oleksii',
+      gender: 'male',
+      legs: 2,
+      hands: 2,
+      saying: 'Wanna be frontend ninja in future!',
+      friends: ['Alex', 'Victoria', 'every lovely kottan on the course']
+   },
+   {
+      species: 'human',
+      name: 'Victoria',
+      gender: 'female',
+      legs: 2,
+      hands: 2,
+      saying: 'When will you bring the money to home, honey? ',
+      friends: ['Natasha', 'Katya', 'Lada']
+   },
+   {
+      species: 'catWoman',
+      name: 'Anjela',
+      gender: 'female',
+      legs: 2,
+      hands: 2,
+      saying : catsPhrase,
+      friends: ['definitely nobody except her own slaves, she\'s a cat, you know...']
+   }
+]
+
+
+   function formOutput(obj) {
+
+      const outputArr = [obj.species, obj.name, obj.gender, obj.legs, obj.hands, obj.saying, obj.friends];
+
+      for (let i = 0; i < outputArr.length; i++) {
+         if (Array.isArray(outputArr[i])) {
+            outputArr[i] = outputArr[i].join(', ')
+         }
+      }
+
+      const result = outputArr.join('; ');
+      
+      return result
```

Comment:
(1) As a lazy peer developer tasked to add 20 more properties I'd like to those props names without need to write `obj.` every time.

(2) We can `.map` through array of initial data, whatever it is, converting arrays to strings

(3) As after (2) we have only strings and numbers in our array we can chain conversion `.map` with `.join`

(4) ...and finally we can have a single expression and do not need `result` variable.